javascript/node/selenium-webdriver/test/cookie_test.js

```diff
@@ -212,6 +212,53 @@ suite(function (env) {
         assert.equal(actualCookie.sameSite, 'Lax')
       }
     )
+
+    ignore(env.browsers(Browser.FIREFOX, Browser.IE, Browser.SAFARI)).it(
+      'can add same site cookie property to `None` when cookie is Secure',
+      async function () {
+        let cookie = createSameSiteCookieSpec('None', {
+          secure: true,
+        })
+        let childUrl = fileserver.whereIs('child/childPage.html')
+        await driver.get(childUrl)
+        await driver.manage().addCookie(cookie)
+        await assert.doesNotReject(
+          async () => await driver.manage().addCookie(cookie)
+        )
```

Comment:
In case of `None`, the cookie has to be `secure`, and I found this means the `getCookie` call would not return the cookie (since the test page in localhost is not served via `https` I presume?). 

I figured the best way to test this is to test that id does *not* throw an exception. In fact, an invalid combination (see below tests) would cause the underlying driver to raise an exception. 
This sounds sufficient to me but let me know if this is not enough.

Reply:
I think that this is fine